client/scripts/generate-regions-json.js

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs'
+
+const DATA_REGION_FILE = 'data/regions.json'
+const REGIONS_LIST_PATH = 'node_modules/caniuse-lite/data/regions'
+
+const regions = {
+  continents: {
+    'alt-ww': 'Global',
+    'alt-af': 'Africa',
+    'alt-an': 'Antarctica',
+    'alt-as': 'Asia',
+    'alt-eu': 'Europe',
+    'alt-na': 'North America',
+    'alt-oc': 'Oceania',
+    'alt-sa': 'South America'
+  },
+  countries: getCaniuseCountries()
+}
+
+fs.writeFileSync(DATA_REGION_FILE, JSON.stringify(regions))
+process.stdout.write(`A file "client/${DATA_REGION_FILE}" with regions has been created`)
```

Comment:
```suggestion
process.stdout.write(`A file "client/${DATA_REGION_FILE}" with regions has been created\n`)
```

`write` doesn’t add `\n` to the end (like `console.log()` does).